gradle.properties

```
@@ -1,22 +1,22 @@
-# Done to increase the memory available to gradle.
-org.gradle.jvmargs=-Xmx2G
+# ~~Done to increase the memory available to gradle.~~ Done to make gradle run BLAZINGLY FAST!!111!1
+org.gradle.jvmargs=-Xmx12G
 
-minecraft_version=1.19.4
-yarn_mappings=1.19.4+build.1
-loader_version=0.14.17
+minecraft_version=1.20
+yarn_mappings=1.20+build.1
+loader_version=0.14.21
 
 mod_version = 3.6.0
```

The version should have been 3.7.0
```suggestion
mod_version = 3.7.0
```
Going to attempt to change it right now